Formula/qfs.rb

```diff
@@ -0,0 +1,101 @@
+class Qfs < Formula
+  desc "High-performance, fault-tolerant, distributed file system and its tools"
+  homepage "https://github.com/quantcast/qfs"
+  url "https://github.com/quantcast/qfs.git"
+  version "1.1.4-1"
+  head "https://github.com/quantcast/qfs.git"
+
+  bottle :unneeded
+
+  depends_on "boost"
+  depends_on "cmake" => :build
+  depends_on "maven" => :build
+  depends_on "openssl" => :build
+  depends_on "wget"
+
+  def install
+    ENV.java_cache
+
+    opoo "Building qfs. This may take some time..."
+    system "make", "build", "hadoop-jars", "BUILD_TYPE=release"
+
+    bin.install "build/release/bin/chunkscrubber"
```

Comment:
The `make install` in qfs unfortunately installs to the `build` directory and not the system directories. That's a decision made upstream unfortunately.

Wouldn't `prefix.install "build/release/bin"` install to the `/usr/local/Cellar/qfs/1.1.4-1` directory instead of the `/usr/local/Cellar/qfs/1.1.4-1/bin` like we want? I just want to make sure I understand what you're asking. Would `bin.install Dir["build/release/bin/*"]` be a better solution?